.github/workflows/pr.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
     uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.10
     with:
       enable_check_generated_files: false
+      enable_check_pr_job_dependencies: false
```

Comment:
```suggestion
```

I'd like to avoid switching this to `false` since we typically try not to add optional jobs to our PR workflows.

The `devcontainer` job is an exception to this rule.

Can you add a new input called `optional_jobs` that accepts a multi-line string of jobs that should not be considered required?

usage example:

```yaml
uses: rapidsai/shared-action-workflows/.github/workflows/checks.yaml@branch-23.10
    with:
      enable_check_generated_files: false
      optional_jobs: |
        devcontainer
```